### ISSUE TITLE
Failed to Install 'AnimalTracker"

### ISSUE THREAD

**Issue by arispes** [mathedjoe/animaltracker#62]
Below is the error I receive after running the following lines:
install.packages("devtools")
library(devtools)
install_github("mathedjoe/animaltracker")
library(animaltracker)
run_shiny_animaltracker()

Error: Failed to install 'animaltracker' from GitHub:
  ERROR: failed to lock directory ‘/Library/Frameworks/R.framework/Versions/3.6/Resources/library’ for modifying
Try removing ‘/Library/Frameworks/R.framework/Versions/3.6/Resources/library/00LOCK’

**Comment by mathedjoe** [comment]
Thanks Sergio! We'll look into this now. In the meantime, it may help to
try finding and deleting the 00LOCK file using the path listed in your
error message. (I've seen similar messages in the past, and got them to go
away by deleting the "00LOCK" file from my machine.

Joe

**Comment by arispes** [comment]
Joe, I got it working on my desktop.  Apparently, the mass package update was the problem.  I installed the updated CRAN packages (BH, farmer, yams, fun, tidyselect, has, pretty units, DBI) individually and I didn't get an error.  Processing them all at once generated the problem.

![Screen Shot 2020-02-25 at 10 18 48 AM](https://user-images.githubusercontent.com/45365827/75270701-3a2f5180-57b8-11ea-99c4-d7243c4e0cc6.png)
